particles_0_6_0/Demo/RestaurantsDemo.recipes

```diff
@@ -24,23 +24,32 @@ recipe RestaurantsDemo
     location = location
     restaurants = restaurants
   SelectableTiles
-    consume root
-      provide annotation as annotation
+    consume root #tiles
+      provide tile as tileSlot // must // set
```

Comment:
Are these comments necessary? Why are they here?

Reply:
removed comments and superfluous slot bindings.